content/dev-cluster/sqlserver.tf

```diff
@@ -0,0 +1,91 @@
+resource "kubernetes_deployment_v1" "mssql_deployment" {
+  metadata {
+    name      = "mssql"
+    namespace = local.k8s_shared_namespace
+    labels = {
+      app = "mssql"
+    }
+  }
+  spec {
+    selector {
+      match_labels = {
+        app = "mssql"
+      }
+    }
+    template {
+      metadata {
+        labels = {
+          app = "mssql"
+        }
+      }
+      spec {
+        container {
+          env {
+            name  = "ACCEPT_EULA"
+            value = "Y"
+          }
+          env {
+            name  = "SA_PASSWORD"
+            value = "Passw12#"
+          }
+          image = "mcr.microsoft.com/mssql/server:latest"
```

Comment:
Maybe we should switch to `2022-latest` here as well, update `SA_PASSWORD` and add collation variable?